draft-kuehlewind-iab-rfc4053bis.md

```diff
@@ -203,9 +203,7 @@ liaison statements. There are no requirements from the IETF on the format
 of the actual liaison statement, however, we require
 the metadata (address information and purpose) as indicated in the previous
 section to be recorded explicitly. As such when receiving statement from other organisation
-these metadata should be extracted. If the content of the
-statement is not sent in plain text, the plain text body field may
-be empty and the received laision statement is uploaded as an attachment.
+these metadata should be extracted.
```

Comment:
```suggestion
these metadata should be extracted. Statements might be recorded containing only a plain text body,
or the body of the statement might only be found in the original received attachment.
```
Not fully sure we need this, but it might still be useful to say somehow that the body of an attachment does not need to be extracted and converted into plain text...?